### PR TITLE
Bump ssl versions

### PR DESCRIPTION
**Pull request by bog-dan-ro**
1.1.1t -> 1.1.1u
 3.1.0 -> 3.1.1

Use 21.4.7075529 for ssl 1.x

Fixed: #47

### CODE REVIEW

**Comment by bog-dan-ro** [comment]
@Issam-b please help with review.

**Comment by Issam-b** [comment]
I'll take a look soon

**Comment by bog-dan-ro** [comment]
@Issam-b any joy ? I just got the confirmation that it fixed  #47 ;-).

**Comment by Issam-b** [comment]
@bog-dan-ro did you see my comment above?

**Comment by bog-dan-ro** [comment]
> @bog-dan-ro did you see my comment above?

This `I'll take a look soon` one ?

**Review by Issam-b — COMMENTED** [review]
I might've forgot to actually commit the comment, if that change makes sense it's good to do it to have less conflict with the other pull request.

File: build_ssl.sh [hunk 1]
```diff
@@ -62,10 +62,10 @@ configure_ssl() {
     done
 
     case $ssl_version in
-    1.1.1t)
+    1.1.1*)
```
[the remark this answers is not in the document]
Comment (bog-dan-ro) [reply build_ssl.sh:65]: The 5.13.2 issue was caused by the ndk version, for 1.1 we need to use an older version (e.g. 21).

[the remark this answers is not in the document]
Comment (Issam-b) [reply build_ssl.sh:65]: this works for me anyway, let's go with it.

**Pull request merged**